sig-api-machinery/annual-report-2021.md

```diff
@@ -4,15 +4,15 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Server Side Apply promoted to GA
+   - Deprecated v1beta1 of Custom Resources and Webhooks (in favor of GA version)
+   - API Priority and Fairness introduced v1beta2
+   - A massive amount of improvements in the API Expression WG
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - We are evaluating long term the potential benefits that generics in go1.19 could provide.
```

Comment:
Thank you for mentioning this @fedebongio, where is this being tracked and discussed so that folks interested in offering a hand can follow along?